packages/web-components/src/horizontalbarchart/horizontalbarchart.template.ts

```diff
@@ -15,7 +15,7 @@ getGroupper(tabsterCore);
  */
 export function horizontalbarchartTemplate<T extends HorizontalBarChart>(): ElementViewTemplate<T> {
   return html<T>` <template>
-    <div ${ref('rootDiv')} class="rootDiv">
+    <div ${ref('rootDiv')} class="root-div">
```

Comment:
You don’t even need this wrapping element. You can target the host binding element in your style sheet with the ‘:host’ selector

Reply:
Applied these changes in https://github.com/microsoft/fluentui/pull/33215